tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/SourceCodeIntegrationGitMetadataTests.cs

```diff
@@ -24,7 +24,7 @@ public SourceCodeIntegrationGitMetadataTests(ITestOutputHelper output)
     [Trait("RunOnWindows", "True")]
     public async Task ManualAndAutomatic()
     {
-        const int expectedSpans = 36;
+        const int expectedSpans = 37;
```

Comment:
Not a fan of this test re-using the sample and duplicating this - I'd rather just move the extra assertions to `ManualInstrumentationTests` - will do that in a separate PR